README.md

```diff
@@ -208,7 +210,7 @@ ship.setCustomData("AGC_ApplyCustomOptions", mapOf("!MAGIC!Missile" to listOf("F
 
 ```java
 // Java
-ship.setCustomData("AGC_ApplyCustomOptions", Collections.singletonMap("!MAGIC!Missile", Arrays.asList("ForceAF", "NoFighters")));
+// ship.setCustomData("AGC_ApplyCustomOptions", Collections.singletonMap("!MAGIC!Missile", Arrays.asList("ForceAF", "NoFighters")));
```

Comment:
This was bugging out for me, lemme know if it isn't supposed to be commented out

Reply:
Nope, this wasn't supposed to be commented out :P